.travis.yml

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
Should this be ignored?

Reply:
Yeah, as we don't use travis CI it's pointless committing it.